.github/workflows/crowdin.yml

```
@@ -0,0 +1,35 @@
+name: Crowdin Action
+
+on:
+  push:
+    branches: [ develop ]
+    
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  synchronize-with-crowdin:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Crowdin action
+        uses: crowdin/github-action@1.0.4
```

we should fork this action under `wireapp` organization to prevent any chnage of the action
```suggestion
        uses: wireapp/github-action@1.0.4
```

---

Done